verification/verify.go

```diff
@@ -57,6 +58,9 @@ type Params struct {
 
 	// Cached data when local object storage is used
 	Renditions [][]byte
+
+	// External object storage used by broadcaster (can be defined per-stream)
+	OS drivers.OSSession
```

Comment:
Why do we need this? URIs should already contain locations of the renditions when data is in an external OS.

Reply:
But in verifier we need to distinguish between local OS and external OS

https://github.com/livepeer/go-livepeer/blob/0e3578d322ec397926f0eea9c63444ecf8e2a4e5/verification/epic.go#L191

Reply:
Ah so we removed drivers.IsOwnExternal and now we call IsExternal() and IsOwn() on each driver session.